source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -325,6 +325,12 @@ class CarParkSerializer(PbNestedSerializer):
     available_PRM = jsonschema.IntField()
     occupied_PRM = jsonschema.IntField()
     total_places = jsonschema.IntField()
+    # items for new api star
+    available_ridesharing = jsonschema.IntField()
```

Comment:
this fields are mandatories for a CarPark ?

Reply:
Yes, for new interface